test/zdtm/static/socket_udp-corked.c

```diff
@@ -22,19 +23,22 @@ const char *test_author = "Pavel Emelyanov <xemul@parallels.com<>\n";
 #include <netinet/udp.h>
 
 static int port = 8880;
+static char buf[11];
 
 #define MSG1 "msg1"
+#define MSG2 "msg_2"
+#define MSG "msg1\0msg_2"
 
 int main(int argc, char **argv)
 {
-	int ret, sk1;
+	int ret, sk1, sk2;
 	socklen_t len = sizeof(struct sockaddr_in);
-	struct sockaddr_in addr1;
+	struct sockaddr_in addr1, addr2, addr;
 	int opt;
 
 	test_init(argc, argv);
 
-	sk1 = socket(PF_INET, SOCK_DGRAM, IPPROTO_UDP);
+	sk1 = socket(PF_INET, SOCK_DGRAM, IPPROTO_UDP); // server udp
```

Comment:
Hmm, here we add a comment to explain what is `sk1`.
What do you think about renaming the variable from `sk1` to something like `server_sk` (and `sk2` to `client_sk`) instead?
... we could do the same for `addr1` and `addr2`.

Reply:
Yes, I think that would be more self-explained. I just try to follow the naming style with the other socket tests in the first place.  Can do with addr1 and addr2 as well.